src/android/toga_android/window.py

```diff
@@ -1,5 +1,10 @@
 from . import dialogs
 
+from rubicon.java import JavaClass
+Intent = JavaClass("android/content/Intent")
+Activity = JavaClass("android/app/Activity")
+Uri = JavaClass("android/net/Uri")
```

Comment:
These declarations should be moved to toga_android.libs so they aren't accidentally declared multiple times.

Reply:
Yes, I saw that there are lots of such declarations in libs/android_widgets.py
I just didn't know whether you wanted to have them there because my declarations are also not widgets...

Reply:
That's a good point. I'd go even simpler - just call it `android.py`, and then organise the internals of the file by Java class structure - i.e., put in (alphabetically ordered) sections for `android.app.*`, `android.content.*`, `android.widget.*`, and so on. That will end up with one very large file - but it will be easier to determine if a class has already been wrapped.

Reply:
Should I create a new file libs/android.py with  just my classes or should I rename the existing libs/android_widget.py?
Renaming sounds dangerous. Wouldn't that break existing code?

Reply:
I now created a new file with only my declaration.
I'll leave it to you whether you want to merge the two files...and take the responsibility that everything still works afterwards :-)